devops/pipeline/e2etest.yaml

```diff
@@ -1,6 +1,9 @@
 # E2E test pipeline - performs end-to-end tests on the following platforms
+#  - Debian 9
 #  - Ubuntu 18.04
 #  - Ubuntu 20.04
+# 
+# See "!!! ADD NEW DISTROS HERE !!!" for new distro placeholder
 
```

Comment:
Is this comment really necessary?

If it is, please remove the many ! and change to lowerase

Reply:
Its meant to be a placeholder to we can just find the different placeholder locations and add the new distro into those placeholders